pkg/vz/vz_driver_darwin.go

```diff
@@ -73,8 +73,9 @@ func (l *LimaVzDriver) Validate() error {
 		"AdditionalDisks",
 		"Audio",
 		"Video",
+		"OS",
 	); len(unknown) > 0 {
-		logrus.Warnf("Ignoring: vmType %s: %+v", *l.Yaml.VMType, unknown)
+		logrus.Warnf("Ignoring following configuration due to the use of vmType %s: %+v", *l.Yaml.VMType, unknown)
```

Comment:
```suggestion
		logrus.Warnf("vmType %s: Ignoring %+v", *l.Yaml.VMType, unknown)
```

Reply:
Resolved in [1f2c1e2](https://github.com/lima-vm/lima/pull/1769/commits/1f2c1e2ff9363f466d8f743458270f71605301f6)

Reply:
This is obviously better than the original. I totally accept it. However, forgive my wasting of time for such an irrelevant issue, I am still concern this is not clear enough for folks not familiar with it (such as the person i was days ago). Personally I'd like to make it more verbose. How do you like it?

Reply:
> Personally I'd like to make it more verbose.

How will it look like?

Reply:
Perhaps like `Prefer vmType %s: Ignoring ...` or `Ignoring due to vmType %s: ...`? I am not sure